src/app/components/ResultsCount/ResultsCount.jsx

```diff
@@ -67,7 +88,16 @@ class ResultsCount extends React.Component {
     if (count !== 0) {
       return (<h2>Displaying {currentResultDisplay} of {countF} results {displayContext}</h2>);
     }
-    return (<h2>No results found. Please try another search.</h2>);
+
+    if (this.checkSelectedFilters()) {
+      return (
+        <h2>No results for the keyword "{searchKeywords}" with the chosen filters. Try
```

Comment:
Sometimes there are multiple keywords. Should the message reflect that by reading "keywords" when appropriate?

Reply:
I think so. I'll make the update.